src/Hero-sec/Hero.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import bgImg from '../Images/img-1.png';
+import menfeat from '../Images/bg.jpg';
+import femfeat from '../Images/womenspo.jpg';
+function Hero() {
+  return (
+    <div>
+      <div
+        className='md:flex justify-between items-center  
+        w-full h-4/6'
+      >
+        <div className='grid md:grid-cols-2 max-w-[1240px] m-auto'>
```

Comment:
Could help if the images fill up more? 
A broad comment could be that the site looks a little empty. Use your real estate, there's no FSI ! 
just on opinion, doesnt break functionality much 
![image](https://user-images.githubusercontent.com/75421090/161649773-dca5919a-109e-4b60-9095-d592d26f44d7.png)